Cargo.toml

```diff
@@ -1,42 +1,7 @@
 [workspace]
 members = [
-    "crates/gitbutler-tauri",
-    "crates/gitbutler-git",
-    "crates/gitbutler-watcher",
-    "crates/gitbutler-watcher/vendor/debouncer",
-    "crates/gitbutler-testsupport",
-    "crates/gitbutler-cli",
-    "crates/gitbutler-branch-actions",
-    "crates/gitbutler-sync",
-    "crates/gitbutler-oplog",
-    "crates/gitbutler-repo",
-    "crates/gitbutler-repo-actions",
-    "crates/gitbutler-command-context",
-    "crates/gitbutler-feedback",
-    "crates/gitbutler-config",
-    "crates/gitbutler-project",
-    "crates/gitbutler-user",
-    "crates/gitbutler-branch",
-    "crates/gitbutler-reference",
-    "crates/gitbutler-error",
-    "crates/gitbutler-serde",
-    "crates/gitbutler-secret",
-    "crates/gitbutler-id",
-    "crates/gitbutler-storage",
-    "crates/gitbutler-fs",
-    "crates/gitbutler-time",
-    "crates/gitbutler-commit",
-    "crates/gitbutler-tagged-string",
-    "crates/gitbutler-url",
-    "crates/gitbutler-diff",
-    "crates/gitbutler-operating-modes",
-    "crates/gitbutler-edit-mode",
-    "crates/gitbutler-cherry-pick",
-    "crates/gitbutler-oxidize",
-    "crates/gitbutler-stack",
-    "crates/gitbutler-hunk-dependency",
-    "crates/gitbutler-settings",
-    "crates/gitbutler-workspace",
+    "crates/gitbutler-*",
+    "crates/but-*"
```

Comment:
Oh my days! This is so much better

Reply:
I did it to highlight the two-class society we are building :D.